targeteditor/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -37,11 +54,54 @@ final case class AladinContainer(
 object AladinContainer {
   type Props = AladinContainer
 
+  /**
+    * On the state we keep the svg to avoid recalculations during panning
+    */
+  @Lenses
+  final case class State(svg: Option[Svg], fov: Fov, current: Coordinates)
+
+  object State {
+    val Zero: State = State(None, Fov(Angle.Angle0, Angle.Angle0), Coordinates.Zero)
+  }
+
+  // TODO: We may want to move these to gsp-math
```

Comment:
We should also probably unify with how it's shown in the elevation plots. Right now, in the plots I'm showing with the degrees, seconds and minutes symbols. What's the usual format for angles?

Reply:
In this case they are not really angles, they are coordinates so using : seems right
FOV is special as we want to show in degrees if it is a large FOV while going down to mins/seconds/etc when they get smaller